src/main/java/com/uid2/admin/job/jobsync/KeyAclSyncJob.java

```diff
@@ -0,0 +1,76 @@
+package com.uid2.admin.job.jobsync;
+
+import com.uid2.admin.job.model.Job;
+import com.uid2.admin.model.PrivateSiteDataMap;
+import com.uid2.admin.store.KeyAclStoreFactory;
+import com.uid2.admin.util.PrivateSiteUtil;
+import com.uid2.shared.auth.EncryptionKeyAcl;
+import com.uid2.shared.auth.OperatorKey;
+import com.uid2.shared.store.reader.RotatingKeyAclProvider;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class KeyAclSyncJob implements Job {
+    private final Collection<OperatorKey> globalOperators;
+    private final Map<Integer, EncryptionKeyAcl> globalAcls;
+    private final KeyAclStoreFactory factory;
+
+    public KeyAclSyncJob(KeyAclStoreFactory factory,
+                         Collection<OperatorKey> globalOperators,
+                         Map<Integer, EncryptionKeyAcl> globalAcls) {
+        this.factory = factory;
+        this.globalOperators = globalOperators;
+        this.globalAcls = globalAcls;
+    }
+
+    @Override
+    public String getId() {
+        return "global-to-site-scope-sync-keyAcls";
+    }
+
+    @Override
+    public void execute() throws Exception {
+        HashMap<Integer, Map<Integer, EncryptionKeyAcl>> desiredState = PrivateSiteUtil.getEncryptionKeyAcls2(globalOperators, globalAcls);
```

Comment:
com.uid2.admin.store.writer.KeyAclStoreWriter.upload takes Map<Integer, EncryptionKeyAcl> data so i think PrivateSiteUtil method needs to be changed?

Reply:
ok as discussed with return `HashMap<Integer, Map<Integer, EncryptionKeyAcl>>` for now